### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jnthnvctr**

### CODE REVIEW

**Comment by jnthnvctr** [comment]
Minor wording changes, moving some words around.

File: b288702/README.md [hunk 1]
```diff
@@ -1,4 +1,28 @@
...
+5. Window-PoSt-32GiB (Poseidon)
+6. Window-PoSt-64GiB (Poseidon)
+
+### Participation Requirements
+
+The following are requirements for participating in Filecoin's mainnet Phase 2. Note that runtime varies inversely with the number of CPU threads available.
+
+| Circuit            | RAM Req. | Disk Space Req. | Runtime 96 CPU Threads | Runtime 8 CPU Threads |
+| :------------------ | :--------: | :---------------: | :-----------------------: | :---------------------: |
+| SDR-PoRep-32GiB    | 6GB      | 52GB            | 45m                    | 6h                    |
```
Comment (vmx) [inline b288702/README.md:18]: I don't think these numbers are correct. My machine ran out of RAM and I have 32GiB (I had other things running, but I needed more than 6GiB).

Also the disk space doesn't seem to be correct. The phase 1.5 parameters are alone >70GiB.

File: b288702/README.md [hunk 2]
```diff
@@ -18,7 +42,7 @@ $ curl -O https://raw.githubusercontent.com/filecoin-project/phase2-attestations
 
 3. Verify Phase2 contributions:
 
-* Running `./verify_all winning 32` will verify all Phase 2 contributions made to the Winning-PoSt-32GiB circuit's Groth16 parameters and check that the verified Groth16 parameters are being used by the Filecoin network.
+In order to run the verifcation, you will need to download the relevant Phase 1 file for the circuit and run the verification script for that specific circuit. The verification script will verify all Phase 2 contributions made to the Groth16 parameters for a circuit of that specific size and check that the verified Groth16 parameters are being used by the Filecoin network. The commands and associated Phase 1 file for each circuit are listed below.
```
Comment (porcuquine) [inline b288702/README.md:45]: Should be 'verification', not 'verifcation'.

File: b288702/README.md (hunk 1 above)
Comment (jnthnvctr) [reply b288702/README.md:18]: oh wait, yeah this is wrong - I think this is the contribution req.

**Pull request merged**